packages/validator/README.md

```diff
@@ -805,7 +805,7 @@ Keyword `__insert` works similarly to `__copy`, except it doesn't copy a value o
 
 ### Target path
 
-`__target` in actions specifies path to parameter that will be modified, there can be an array in this path, to deal with this path can contain `[]` which will push back new item into the array and use it in the target, or path can use `[_]` which points to last item in the array. Parameter from path where the action is located can be accessed via `{{X}}`, where X is position of the parameter in the path numbered from 0. Target path can also include keyword `__all`, in which the action will be performed on every child of previous element.
+`__target` in actions specifies path to parameter that will be modified, there can be an array in this path, to deal with this path can contain `[]` which will push back new item into the array and use it in the target, or path can use `[_]` which points to last item in the object. Parameter from path where the action is located can be accessed via `{{X}}`, where X is position of the parameter in the path numbered from 0. Target path can also include keyword `__all`, in which the action will be performed on every child of previous element.
```

Comment:
I'm not sure about this change, because in this case it is literally referring to last object in array. So if in the specification would be some thing like `[ { param1: "value" }, { param2: "value" } ]`, then `[_]` would refer to `{ param2: "value" }`, which is last item in the array not object.

Reply:
So `[_]` can refer to both the last object in an array or an object?

Reply:
It should be array only, is there any inconsistency somewhere, that implies it can be also an object?

Reply:
nooo ... now i'm back at my original confusion from yesterday: if `[_]` refers/points to the last index in an array, why does this action in the last example:
```
"__insert": {
	"__target": "array[_].parameter",
	"__value": {}
}
```
result in `parameter` being inserted into *every* object in `array` instead of just the last object (i.e. the object containing `param2`)?